common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/PushBlockStream.java

```diff
@@ -85,12 +96,13 @@ public boolean equals(Object other) {
 
   @Override
   public int encodedLength() {
-    return Encoders.Strings.encodedLength(appId) + 16;
+    return Encoders.Strings.encodedLength(appId) + 20;
```

Comment:
nit: may be 16 + 4 similar to the other place or 4 + 4 + 4 + 4 + 4?.

Reply:
Updated